yazi-plugin/Cargo.toml

```diff
@@ -39,6 +39,7 @@ tokio         = { version = "1.38.0", features = [ "full" ] }
 tokio-stream  = "0.1.15"
 tokio-util    = "0.7.11"
 unicode-width = "0.1.13"
+textwrap	  = "0.14.2"
```

Comment:
I want to avoid introducing new crates as much as possible. Is it possible to write a simple function to handle line breaks? 

We can use the [`unicode-width`](https://crates.io/crates/unicode-width) crate to get the character width and it's already in the dependency tree.

Reply:
I tried to push ChatGPT to do that but after about 50 iterations concluded that it's pretty hard due to various edge cases. This crate solved the issue momentarily.

Although, I admit that probably my Rust skills or the lack thereof has considerably contributed to the difficulty :)